### ISSUE TITLE
(How do I reopen this issue) Not able to import XCCDF results 

### ISSUE THREAD

**Issue by ronaldsimmons** [NUWCDIVNPT/stig-manager#890]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest version of STIG Manager?

- [ ] I am using the latest [Release](https://github.com/NUWCDIVNPT/stig-manager/releases).
- [X] I am NOT using the latest [Release](https://github.com/NUWCDIVNPT/stig-manager/releases). I am aware that the first thing I will be asked to do is update the application so that I have the latest bugfixes.

### Where are you experiencing the issue?

- [ ] API
- [X] UI or other client
- [ ] Deployment
- [ ] Elsewhere

### Current Behavior

When importing XCCDF results I receive the error no benchmark element. not sure if this is a permissions issue or the fact that they have not imported the benchmark files to the library.

I have verified 
1. the FQDN, IP and MAC address are exact matches between the export and the asset
2. verified the XCCDF by importing it to STIG workbooks exported from STIG manager and imported to STIG viewer

### Expected Behavior

successful XCCDF import

### Can you provide screenshots, logs, or other useful artifacts?

![Screenshot 2023-01-09 152415](https://user-images.githubusercontent.com/122317213/211411223-64eba35f-7329-412b-a47e-3376e65c7a20.jpg)
![Screenshot 2023-01-09 152137](https://user-images.githubusercontent.com/122317213/211411228-a27c4177-7632-4d37-bd2e-e330f2565250.jpg)

### Describe your Environment

```markdown
- Hosting: Provided by Cloud One
- Browser: Microsoft Edge
- OS: Windows 11
- Node:
- npm:
```

**Comment by cd-rite** [comment]
Hi @ronaldsimmons Our current implementation requires that the Benchmark element be present in the XCCDF file to identify the STIG being imported. 
The tools I've seen can all produce XCCDF files with a proper Benchmark element. What tool are you using that produced results without it? Are you able to provide the XCCDF file you are trying to import?
I'm not really following your process from item 2 "verified the XCCDF by importing it to STIG workbooks exported from STIG manager and imported to STIG viewer. " Can you clarify a bit?
Is STIG Workbooks a tool? Which results did you export from STIG Manager, if you can't get them in there?  If STIG Viewer exported the results, it is most likely a checklist not in XCCDF format, and should have the .ckl extension, not .xml

The app also requires that the DISA STIG has already been imported into the Library. (The exact revision does not necessarily need to match, just the STIG ID. However, no Reviews will be imported for Rule IDs that are not known to the STIGMan system.) 

So right now, I'd say the app is working as designed. Perhaps a setting in whatever tool you are using for XCCDF results can have it create a file with the proper Benchmark elements?
Feel free to reopen this issue with more information. 
Thanks!

**Comment by ronaldsimmons** [comment]
Thank you for your response. I can't provide a copy of the xccdf as they are ACAS scan results of active DISA systems, When I said I verified the issue I meant I tried to import multiple SCAP scan results and they all produced the same results "no Benchmark Elements" I know the system owner has loaded the appropriate STIGS to the library but has not the DISA benchmarks.

**Comment by ronaldsimmons** [comment]
@cd-rite How do I reopen this issue?

**Comment by ronaldsimmons** [comment]
to further troubleshoot I performed SCAP scans with Trellix Formerly McAfee Policy Auditor again I was able to import the xccdf scap results into a STIG in DISA STIG Viewer 2.17 but not STIG Manager 1.3.0

**Comment by csmig** [comment]
STIG Manager currently only parses XCCDF result files with a root-level `<Benchmark>` element. This is the format used by SCC and OpenSCAP, the tools we currently target. The error messge you posted suggests ACAS generates a less common XCCDF dialect with a root-level `<TestResult>` element and no `<Benchmark>`.

Please review your files and let us know if this explanation is correct. If so, then this issue should be re-opened as a feature request and not a bug.

**Comment by ronaldsimmons** [comment]
I understand what you are saying and I apologize but I was able to import them to a STIG manager deployment that we had set up earlier and now we have moved the service to one we don't manage and that is when we lost the ability to import these files. the only difference I can see is they did not load the benchmark file to the STIG library. they only loaded STIGS

**Comment by ronaldsimmons** [comment]
I can send you a sanitized xccdf if you need it

**Comment by csmig** [comment]
Yes, posting a sanitized file that fails to import will be helpful. We do not have access to an ACAS. 

The error message you're receiving is generated in the web app very early in the parsing process. It should only be thrown when the root element of the XCCDF file is not `<Benchmark>`.

**Comment by ronaldsimmons** [comment]
OK I am providing two sanitized. SCAP scan xccdf xml files one from ACAS (Tenable) and the other from Trellix (McAfee) HBSS both import into the DISA STIG viewer but have the same error when importing to the STIG Manager. I changed the extension from .xml to .txt for transit.

[Trellix_HBSS_(McAfee)Export.txt](https://github.com/NUWCDIVNPT/stig-manager/files/10828859/Trellix_HBSS_.McAfee.Export.txt)
[Tenable_ACAS_Export.txt](https://github.com/NUWCDIVNPT/stig-manager/files/10828860/Tenable_ACAS_Export.txt)

**Comment by csmig** [comment]
Those files are helpful, thank you. They both have a root element `<TestResult>` instead of `<Benchmark>` which is unsupported in all released versions of STIG Manager to date. However, since they represent a compliant variant of XCCDF and since we now have example files we will soon add support in both STIG Manager and STIGMAN Watcher.

**Comment by ronaldsimmons** [comment]
thank you for helping with this

**Comment by ronaldsimmons** [comment]
So what does the system owner need to do to get STIG Manager to work properly

**Comment by ronaldsimmons** [comment]
I am not the system owner of the STIG Manager, what do they need to do to pull the update? I may need to open a ticket with them and tell them what they need to do

**Comment by ronaldsimmons** [comment]
@cd-rite I know the issue has been resolved but I need to know what to tell the system owner to do to pull the updates
